.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
```

Comment:
I would remove these lines to clean up file... these seem to be out the box suggestions from circle